index.js

```diff
@@ -455,21 +455,15 @@ function fastifyMultipart (fastify, options = {}, done) {
       try {
         await pump(file.file, target)
         requestFiles.push({ ...file, filepath })
+        this.tmpUploads.push(filepath)
         // busboy set truncated to true when the configured file size limit was reached
         if (file.file.truncated) {
           const err = new RequestFileTooLargeError()
           err.part = file
           throw err
         }
-        // Delay adding the filepath to tmpUploads until the file has been checked for size
-        this.tmpUploads.push(filepath)
       } catch (err) {
-        try {
-          await unlink(filepath)
-        } catch (err) {
-          this.log.debug({ err }, 'could not delete file')
-        }
-
+        this.log.error(err)
```

Comment:
Adding a bit more context makes it easier to know where it failed.

```js
this.log.debug({ err }, 'save request file')
```

Reply:
Done. Are the object brackets necessary? I added them just in case

Reply:
No, both should work.